cmd/ack-generate/command/common.go

```diff
@@ -250,6 +259,11 @@ func loadModel(svcAlias string, apiVersion string) (*ackmodel.Model, error) {
 			return nil, fmt.Errorf("service %s not found", svcAlias)
 		}
 	}
+
+	if cpAPIGroupSuffix != "" {
+		sdkAPI.APIGroupSuffix = cpAPIGroupSuffix
+	}
```

Comment:
Instead of the conditionals above, how about adding new `apiGroup` and `defaultCfg` parameters to the `loadModel` function?

Reply:
Yeah, this is the best approach.  I think I didn't consider this before because I was trying to touch as few places as possible, but it turns out only one other package uses this function outside the ones I've already changed.